Objective-C/UserSampleId/README.md

```diff
@@ -0,0 +1,16 @@
+Swrve SDK User Sample Id
+-------------------------
+Example of how to use the user id to generate an int between 1 and 100 and update a user property in Swrve
+
+How to run the demo in XCode
+----------------------------
+- Open the project located under UserSampleIdExample/UserSampleIdExample.xcodproj
+- Replace YOUR_APP_ID in AppDelegate.m with your Swrve app ID.
+- Replace YOUR_API_KEY in AppDelegate.m with your Swrve API key.
+- Run UserSampleIdExample app normally.
+
+Overview
```

Comment:
How to use this in your own app?
--

* Copy blahblah.m and blahblah.h to your own project.
* Call function blah